openbb_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -1821,17 +1805,27 @@ def call_pt(self, other_args: List[str]):
                 console.print(no_ticker_message)
                 return
 
-            business_insider_view.price_target_from_analysts(
-                symbol=self.ticker,
-                data=self.stock,
-                start_date=self.start,
-                limit=ns_parser.limit,
-                raw=ns_parser.raw,
-                export=ns_parser.export,
-                sheet_name=" ".join(ns_parser.sheet_name)
-                if ns_parser.sheet_name
-                else None,
-            )
+            if ns_parser.source == "BusinessInsider":
+                business_insider_view.price_target_from_analysts(
+                    symbol=self.ticker,
+                    data=self.stock,
+                    start_date=self.start,
+                    limit=ns_parser.limit,
+                    raw=ns_parser.raw,
+                    export=ns_parser.export,
+                    sheet_name=" ".join(ns_parser.sheet_name)
+                    if ns_parser.sheet_name
+                    else None,
+                )
+            elif ns_parser.source == "FinancialModelingPrep":
```

Comment:
Wasn't able to test it (special endpoint) - I've just changed a little the message to provide more context.

Reply:
tested now. works like a charm:
![image](https://user-images.githubusercontent.com/48914296/224067407-f12abc15-a17c-42e3-9e07-ad039d0a91eb.png)